### ISSUE TITLE
Euro currency symbol in Markdown file causes "Undefined control sequence" error

### ISSUE THREAD

**Issue by oliver** [Wandmalfarbe/pandoc-latex-template#84]
When trying to create a PDF from a Markdown file containing a € (Euro) currency symbol, I get an "Undefined control sequence" error:
```
$ pandoc -o test_euro_char.pdf --template eisvogel test_euro_char.md 
! Undefined control sequence.
l.273 Inspiration for the \euro

pandoc: Error producing PDF
```

This is the file I was trying to convert:
```
Inspiration for the € symbol itself came from the Greek epsilon.
```

If I don't use the eisvogel template, pandoc successfully creates a PDF from this input.

Based on jgm/pandoc#309 I have added `\usepackage{eurosym}` to eisvogel.latex, and that has fixed the problem for me. But since I have no experience with Latex, I don't know if this problem actually needs to be fixed in eisvogel template or somewhere else.

**Comment by Wandmalfarbe** [comment]
I can't reproduce the issue. This example dcocument compiles without errors:

```markdown
---
title: "Test #64: Euro Symbol"
author: [Example Author]
date: "2018-03-31"
keywords: [Test]
...

Inspiration for the € symbol itself came from the Greek epsilon.
```

```bash
pandoc euro.md -o euro-pdflatex.pdf --from markdown --template eisvogel --listings --pdf-engine pdflatex
pandoc euro.md -o euro-xelatex.pdf  --from markdown --template eisvogel --listings --pdf-engine xelatex
```

What version of pandoc are you using? What is the command you use to compile the document?

**Comment by oliver** [comment]
Interesting! I'm using pandoc 1.16.0.2 (the version that comes with Ubuntu 16.04) and texlive-latex-recommended 2015.20160320-1ubuntu0.1.

The command I'm using is: `pandoc -o test_euro_char.pdf --template eisvogel test_euro_char.md`

The `--pdf-engine pdflatex` option is not supported in this pandoc version; I can use `--latex-engine pdflatex` though, which is also the default and which produces the error. With `--latex-engine xelatex` or `--latex-engine lualatex` it works fine, though!

So if you can't reproduce this, I'm going to assume that this is caused by my old pandoc version. A possible workaround would be to use `--latex-engine xelatex` or to manually modify eisvogel.latex, both of which are good enough for me. From my point of view you can close this issue. Thanks for looking into this, and for making the Eisvogel template available!

**Comment by Wandmalfarbe** [comment]
> The --pdf-engine pdflatex option is not supported in this pandoc version; I can use --latex-engine pdflatex though, which is also the default and which produces the error. With --latex-engine xelatex or --latex-engine lualatex it works fine, though!

Yes, the command was renamed in https://github.com/mb21/pandoc/commit/1cf6f01b07bacf674d2a6cd83663b206759a2d36 which was released as pandoc 2.0 in October 2017. I suggest you update your pandoc installation :)
